SingularConfig.js

```diff
@@ -5,6 +5,11 @@ export class SingularConfig {
     customUserId;
     singularLinkHandler;
 
+    // SKAN
+    skAdNetworkEnabled;
```

Comment:
Maybe let's add default values in the constructor just in case?

Reply:
which will be True or False?

Reply:
Both false by default, the handler is null by default so no need to set it